client/src/context/post/PostState.js

```diff
@@ -48,8 +53,28 @@ const PostState = (props) => {
     }
   }
 
+  const submitComment = async (body) => {
+    const config = { headers: { 'Content-Type': 'application/json' } }
```

Comment:
you do this a lot where you're setting the headers for the content type, is there a way to default this? I'm sure there is with axios and an axios config file